packages/client/src/gen/coordinator/index.ts

```diff
@@ -2008,6 +2008,64 @@ export interface ClosedCaptionEvent {
    */
   type: string;
 }
+
+/**
+ *
+ * @export
+ * @interface CollectUserFeedbackRequest
+ */
+export interface CollectUserFeedbackRequest {
+  /**
+   *
+   * @type {object}
+   * @memberof CollectUserFeedbackRequest
+   */
+  custom?: object;
```

Comment:
```suggestion
  custom?: { [key:string]: any };
```

Reply:
This is a general issue that still hasn't been resolved in the OpenAPI schema. Can you please change the `object` as suggested?